src/marqo/core/index_management/index_management.py

```diff
@@ -34,23 +42,33 @@ class IndexManagement:
         }
         '''
     )
+    _DEFAULT_QUERY_PROFILE_TEMPLATE = textwrap.dedent(
+        '''
+        <query-profile id="default">
+            <field name="maxHits">1000</field>
+            <field name="maxOffset">10000</field>
+        </query-profile>
+        '''
+    )
 
     def __init__(self, vespa_client: VespaClient):
         self.vespa_client = vespa_client
 
-    def create_settings_schema(self) -> bool:
+    def bootstrap_vespa(self) -> bool:
         """
-        Create the Marqo settings schema if it does not exist.
+        Add Marqo configuration to Vespa application package if an existing Marqo configuration is not detected.
```

Comment:
Not a blocker for this PR, but there is an implied 3rd option - an error in configuring vespa. We can detail such a case here, including expected error

Reply:
Under normal circumstances there won't be an error. We don't normally document unexpected errors. I have a ticket to add custom errors to this in case of a network issue and at that point I'll document that custom error.

Reply:
getting sidetracked from the this PR., but I'd say that network errors are almost "normal circumstances"